src/pc/tracker.rs

```diff
@@ -70,16 +82,20 @@ struct SubxtSessionTracker {
 }
 
 /// An outcome for a dispute
-#[derive(Encode, Decode, Debug, Default, Clone)]
+#[derive(Encode, Decode, Debug, Clone)]
 pub struct DisputesOutcome {
```

Comment:
I would rename this struct, as we have `the real outcome` inside of it. `DisputeProgress` or `DisputeInfo` ?